### PR TITLE
Add different highlight for "current day" and "today".

### PR DESCRIPTION
**Pull request by knight-bubble**
https://community.openproject.org/work_packages/19907

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://hirtie-maxim-fix-19907-highlight-current-date.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://hirtie-maxim-fix-19907-highlight-current-date.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: app/assets/stylesheets/content/_in_place_editing.sass [hunk 1]
```diff
@@ -76,6 +76,8 @@ $inplace-edit--color--very-dark: #cacaca
     & > .inplace-edit--date-range-end-date-picker
       @extend %inline-date-picker-container-position-absolute
       right: 0
+  .ui-datepicker .ui-datepicker-calendar td a.ui-state-active
+    border-color: #008bd0 !important
```
Comment (NobodysNightmare) [inline app/assets/stylesheets/content/_in_place_editing.sass]: I'd rather use the primary color here than an arbitrary color value (which will not change among themes)

Comment (NobodysNightmare) [inline app/assets/stylesheets/content/_in_place_editing.sass]: I don't see the need for the `!important` here, works without for me.

@myabc Are there any rules we have regarding usage of this keyword?
I can only hope for "as few as possible".

@hirtie-maxim Are there special circumstances that make it neccessary?

Comment (myabc) [inline app/assets/stylesheets/content/_in_place_editing.sass]: > @myabc Are there any rules we have regarding usage of this keyword?
> I can only hope for "as few as possible".

Agreed.

**Pull request merged**